Sources/Sentry/include/SentryProfiler+Test.h

```diff
@@ -4,18 +4,26 @@
 
 #if SENTRY_TARGET_PROFILING_SUPPORTED
 
+@class SentryDebugMeta;
+@class SentryId;
 @class SentrySample;
+@class SentryTransaction;
 
 NS_ASSUME_NONNULL_BEGIN
 
 void processBacktrace(const sentry::profiling::Backtrace &backtrace,
     NSMutableDictionary<NSString *, NSMutableDictionary *> *threadMetadata,
     NSMutableDictionary<NSString *, NSDictionary *> *queueMetadata,
-    NSMutableArray<SentrySample *> *samples, NSMutableArray<NSMutableArray<NSNumber *> *> *stacks,
```

Comment:
Yes, it can change for a thread between samples if we find that the metadata is not present on one sample but is present for a subsequent sample on the same thread. This occasionally happens when we can't read the metadata for a thread (unclear why this happens but it tends to happen sometimes for a newly created thread).

Reply:
Cool, thanks for confirming!